x-pack/test/cloud_security_posture_functional/cloud_tests/benchmark_sanity.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['common', 'benchmark']);
+
+  describe('Benchmark Page - Sanity Tests', function () {
+    this.tags(['cloud_security_posture_ui_sanity']);
+    let benchmark: typeof pageObjects.benchmark;
+
+    before(async () => {
+      benchmark = pageObjects.benchmark;
+      await benchmark.navigateToBenchnmarkPage();
+      await benchmark.waitForPluginInitialized();
+    });
+
+    it('Benchmark table exists', async () => {
+      expect(await benchmark.benchmarkPage.doesBenchmarkTableExists());
+    });
+
+    it('Benchmarks count is more than 0', async () => {
+      const benchmarksRows = await benchmark.benchmarkPage.getBenchmarkTableRows();
+      expect(benchmarksRows.length).to.be.greaterThan(0);
+    });
+
+    it('For each benchmark, evaluation and complience are not empty', async () => {
+      const benchmarksRows = await benchmark.benchmarkPage.getBenchmarkTableRows();
+      for (const row of benchmarksRows) {
+        const benchmarkName = await benchmark.benchmarkPage.getCisNameCellData(row);
+        const evaluated = await benchmark.benchmarkPage.getEvaluatedCellData(row);
+        const compliance = await benchmark.benchmarkPage.getComplianceCellData(row);
+        expect(await evaluated).to.not.contain(
+          'Add',
```

Comment:
Would be better to add a test subject and use it here and in the next `expect` so that we could rely on the tests even if the design changes

Reply:
@maxcold, I'm not sure I understand which object should have a test subject. In the current implementation, the `getEvaluatedCellData()`, `getComplianceCellData()`, and `getCisNameCellData()` methods retrieve data from the cells by relying on defined test subjects for each column, such as `benchmark-table-column-evaluated`, `benchmark-table-column-compliance`, and `benchmark-table-column-cis-name`. Could you please provide more details or an example to help clarify your suggestion?

Reply:
Sure, sorry for not being specific. The small problem I see is that you are relying on the text `Add` in your assert. The text can change, let's say the product decides that it is better to have `Enroll more accounts` there or just have a plus sign without any text. A bit safer approach I think is to add a new test subj for the button itself and rely on it, instead of on the text

Reply:
Added a new [task](https://github.com/elastic/security-team/issues/9917) to implement test subjects